### ISSUE TITLE
Nice job!

### ISSUE THREAD

**Issue by sauloonze** [cgarwood/hatouch#18]
Hello,

I would like to have a different approach to deal with the lovelace and .yaml files of homeassistant. I have been developing interfaces on Vuejs and bootstrap 4 and I think this project could be a good base to it. 
Are you still using this project? Did you find issues using the websocket API? I would appreciate your option on this.

Thanks

**Comment by cgarwood** [comment]
I jumped ship over to TileBoard when it was released and am now looking at migrating my tablets over to Lovelace so I can take advantage of all the custom elements others have made.

Feel free to use the code as a base if you want, but its all 3 years old at this point. A lot has changed with Home Assistant and I'm guessing there's been a number of Vue updates since then, so it may be easier to start from scratch 🤷‍♂️

**Comment by sauloonze** [comment]
You are right. The custom elements are really handy on lovelace and redevelop them will take time. I have been using the lovelace from the start. Great tool!

Anyway, I was able to run this project and connect to HA. I had to add ```  ws.send('{"type": "auth","access_token": ""}'); ``` to authenticate but overall It still working. :D

Thanks for the reply!
